codalab/rest/schemas.py

```diff
@@ -220,6 +220,18 @@ class BundleSchema(Schema):
     class Meta:
         type_ = 'bundles'
 
+class BundleStoreSchema(Schema):
+    id = fields.String(validate=validate_uuid, attribute='uuid')
+    uuid = fields.String(attribute='uuid')
+    owner = fields.String(attribute='owner_id')
+    name = fields.String()
+    storage_type = fields.String()
+    storage_format = fields.String(validate=validate.OneOf({'UNCOMPRESSED', 'COMPRESSED_V1'}))
```

Comment:
Let's just do this for now

```suggestion
    storage_format = fields.String()
```

so we only need to maintain the enum list in one place, which is the db schema